spec/dependabot/update_checkers/java_script/npm_and_yarn/version_resolver_spec.rb

```diff
@@ -535,6 +555,26 @@
         end
 
         it { is_expected.to eq(Gem::Version.new("15.6.2")) }
+
+        context "of multiple dependencies" do
+          let(:manifest_fixture_name) { "peer_dependency_multiple.json" }
+          let(:yarn_lock_fixture_name) { "peer_dependency_multiple.lock" }
+          let(:dependency) do
+            Dependabot::Dependency.new(
+              name: "react",
+              version: "0.14.2",
+              package_manager: "npm_and_yarn",
+              requirements: [{
+                file: "package.json",
+                requirement: "0.14.2",
+                groups: ["dependencies"],
+                source: nil
+              }]
+            )
+          end
+
+          pending { is_expected.to eq(Gem::Version.new("0.14.9")) }
```

Comment:
@feelepxyz - weirdly, it doesn't look like the peer dependency checker is finding *any* errors for this case. It should be - updating from `0.14.2` to `16.3.1` generates lots of them when done locally. Any ideas?